neurom/view/view.py

```diff
@@ -25,13 +25,12 @@
 # ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-'''
-Python module of NeuroM to visualize morphologies
-'''
+'''visualize morphologies'''
```

Comment:
Changes have to do with:
* removing kwargs
* removing automatic figure creation: this is a development API, and it's better to give complete control to the person using matplotlib
* remove automatic styling, again, a developer can call this if necessary
* rename functions neuron -> plot_neuron